deploy/crds/operator_v1_amazoncloudintegration_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operator.tigera.io/v1
+kind: AmazonCloudIntegration
+metadata:
+  name: example-amazoncloudintegration
+spec:
+  # Add fields here
+  size: 3
```

Comment:
this dummy data needs to be removed. 

I wonder if we don't even want to check this CR in because unlike the other CRs, there's no such thing as a "default empty" one since it contains so much environment config.

The mockup docs I have for this feature have the user basically creating this CR instead of editing it.

Reply:
We don't actually use these CR files for anything as far as I'm aware. I have updated this though.